remote-build.exp

```diff
@@ -41,13 +41,6 @@ if {[file exists ${conf}]} {
 	source ${conf}
 }
 
-# Define default value of variables if not previously defined
-# Directory used by the script to save its files.
-set_def tray_dir ~/tmp/at-build-tray
-# Directory where AT will be installed (aka DESTDIR).
-set_def destdir "\${HOME}/opt"
-set_def default_timeout 30; # 30 seconds
-
 # Directory where AT is installed
 # Before start the building, all AT installs with the same version of that
 # specified in the parameters are removed
```

Comment:
You could also remove the at_dest declaration as it seems it's not used anywhere.